docs/pages/foundations/messaging/components.js

```diff
@@ -0,0 +1,222 @@
+// @flow strict
+import { Text, Flex, Box, Link } from 'gestalt';
+import { type Node } from 'react';
+import Example from '../../../docs-components/Example.js';
+import SandpackExample from '../../../docs-components/SandpackExample.js';
+import modalExample from '../../../examples/modal/limitActionsExample.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+
+export default function MessagingComponentsPage(): Node {
+  return (
+    <Page title="Messaging components">
+      <PageHeader name="Messaging components" type="guidelines" />
+      <MainSection name="Callout">
+        <Example
+          id="calloutExample"
+          name="Callout example"
+          showCode={false}
+          showHeading={false}
+          defaultCode={`<Callout
+  title='Your tag is not working'
+  message='Data may be outdated because your tag is not sendng information. Fix your tag for the most accurate metrics.'
+  type='error'
+  dismissButton={{
+    accessibilityLabel: 'Dismiss banner',
+    onDismiss: () => {},
+  }}
+  iconAccessibilityLabel="Error"
+  primaryAction={{
+    accessibilityLabel: 'Fix tag',
+    label: 'Fix tag',
+    onClick: () => {},
+  }}
+/>`}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>
+            Banners sit on a page or surface and provide status alerts, information or
```

Comment:
I think for now we should leave this as "Callouts sit.." instead of Banners

Reply:
Or maybe, "Banners like Callouts sit..."

Reply:
Done!